labman/db/configuration_manager.py

```diff
@@ -125,6 +134,16 @@ def _get_main(self, config):
         self.test_environment = config.getboolean('main', 'TEST_ENVIRONMENT')
         self.log_dir = config.get('main', 'LOG_DIR')
 
+        self.certificate_file = config.get('main', 'CERTIFICATE_FILE')
+        if not self.certificate_file:
+            self.certificate_file = join(self.install_dir, 'gui',
+                                         'support_files', 'server.crt')
+
+        self.key_file = config.get('main', 'KEY_FILE')
+        if not self.key_file:
+            self.key_file = join(self.install_dir, 'gui', 'support_files',
```

Comment:
Given that the string "support_files" is hardcoded both here and in labman/labman/db/testing.py (`  path_builder = partial(join, dirname(__file__), 'support_files')`), it would be nice to see this value stored in a constant somewhere.